### PR TITLE
Add Kokoro test script

### PR DESCRIPTION
**Pull request by murgatroid99**
CC @matt-kwong

### CODE REVIEW

**Comment by nicolasnoble** [comment]
cc @matt-kwong

File: kokoro-linux.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Install NVM
```
Comment (nicolasnoble) [inline kokoro-linux.sh]: We'll need the copyright blurb (that's bureaucracy), and #!/bin/sh (that's more about making this work properly).

File: kokoro-linux.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# Install dependencies and link packages together
+npm install
+gulp setup
+
+for version in ${node_versions[@]}
+do
+  nvm install $version
+  # Rebuild libraries and run tests
+  gulp native.test
```
Comment (nicolasnoble) [inline kokoro-linux.sh]: Wouldn't you need to do npm install between each change of node version ?

**Comment by matt-kwong** [comment]
This should work for both Linux and MacOS, right?

File: kokoro-linux.sh (hunk 1 above)
Comment (nicolasnoble) [inline kokoro-linux.sh]: The file you're adding is 644. It needs to be 755.

**Comment by nicolasnoble** [comment]
It should work on both Linux and Mac, yes. Windows might be a bit more tricky right off the bat.

**Comment by murgatroid99** [comment]
I think so

File: kokoro-linux.sh (hunk 1 above)
Comment (murgatroid99) [reply kokoro-linux.sh]: I think I forget that more often than I remember.

File: kokoro-linux.sh (hunk 2 above)
Comment (murgatroid99) [reply kokoro-linux.sh]: As mentioned, that command includes a rebuild, and I think that the only thing that depends on the Node version is the grpc extension build itself.

**Comment by nicolasnoble** [comment]
So, #29 shows that gulp won't work without ES6 support, so this script isn't looking like it's going to work before Node 6.

**Comment by murgatroid99** [comment]
It looks like it's a specific issue with some ES6 features. I might just be able to fix it.

**Comment by nicolasnoble** [comment]
I see, it's a problem in our gulp script files, not with gulp itself. Let's remove node 4 and 5 from that PR for now then, and fix it later.

**Pull request merged**